src/factories/BranchFactory.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/Delegator.sol';
+
+
+// FIXME: remove once this can be imported as a solidty contract
+contract Branch {
+    function initialize(address parentBranch, int256 parentPayoutDistributionHash);
+}
+
+
+contract BranchFactory {
+
+    function createBranch(address controller, address parentBranch, int256 parentPayoutDistributionHash) returns (Branch) {
```

Comment:
`_parenteBranch` should be of type `Branch` and `_controller` should be of type `Controller`.